README.md

```diff
@@ -30,14 +30,16 @@ All features are encoded; documentation [link:] (https://www.bcsc-research.org/d
 Source:
 [Breast Histopathology Images](https://www.kaggle.com/paultimothymooney/breast-histopathology-images/)
 
-The original dataset was formed of 162 whole mount slides of Breast Cancer specimen, 277534 oatches of 50 x 50 images were extracted; 198,738 IDC negative and 78,786 IDC positive. The patients file name is in the format:
+The original dataset was formed of 162 whole mount slides of Breast Cancer specimen, 277534 patches of 50 x 50 images were extracted; 198,738 IDC negative and 78,786 IDC positive. The patients file name is in the format:
     uxXyYclassC.png → 10253idx5
 X and Y are the coordinates where the patch were cropped from and C indicates class: 0 is non-IDC and 1 is IDC
 
 ## Questions ##
 
 * Understanding risk factors distributions to promote early detection practices; breast self exam, clinical breast exam and mammogram 
```

Comment:
Phrase as a question: What are the risk factors that can promote early detection practices? Do certain risk factors warrant a breast self exam, clinical breast exam, or mammogram?